core/loadpoint.go

```diff
@@ -1330,7 +1337,7 @@ func (lp *Loadpoint) boostPower(batteryBoostPower float64) float64 {
 		}
 	}
 
-	res := batteryBoostPower + delta
+	res := batteryBoostPower + delta + lp.site.GetResidualPower()
```

Comment:
Anm: `residualPower` addieren um dessen Offset an der `site` zu neutralisieren.

Reply:
Ich kann dir nicht folgen. Ist das so jetzt richtig? Dann müsste es noch getestet werden :O

Reply:
Ja, sollte nun so richtig sein.
Gestern war leider für mich keine Testmöglichkeit in der Praxis mehr.